.eslintrc.js

```diff
@@ -1,4 +1,10 @@
 module.exports = {
   extends: ['react-app'],
-  ignorePatterns: ['dist', 'build', 'node_modules', '.storybook'],
+  ignorePatterns: [
+    'dist',
+    'build',
+    'node_modules',
+    '.storybook',
+    'storybook-static',
+  ],
```

Comment:
IMO we can use --ignore-path .gitignore instead of ingnorePatterns

Reply:
no that way we should write --ignore-path more than once